### ISSUE TITLE
KeyError: 'file.check_managed_changes' on managed files

### ISSUE THREAD

**Issue by andrejohansson** [saltstack/salt#20256]
When running a state I've started to get KeyError exceptions on managed salt files. I'm running on windows minions.

The exception I get
```
          ID: C:\Program Files (x86)\ReadSoft Software Services AB\ReadSoft New Relic Metrics\AgentConfig.json
    Function: file.managed
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "salt/state.py", line 1529, in call
                File "salt/states/file.py", line 1311, in managed
              KeyError: 'file.check_managed_changes'
     Started: 13:22:48.886000
    Duration: 14.0 ms
     Changes:
```

Minion versions report
```
root@dev-eu-master:/srv/salt# time salt '*' test.versions_report                                 
dev-eu-minion:                                                                                   
               Salt: 2014.7.1                                                                    
             Python: 2.7.5 (default, May 15 2013, 22:44:16) [MSC v.1500 64 bit (AMD64)]          
             Jinja2: 2.7.1                                                                       
           M2Crypto: 0.21.1                                                                      
     msgpack-python: 0.4.2                                                                       
       msgpack-pure: 0.1.3                                                                       
           pycrypto: 2.6                                                                         
            libnacl: 1.4.0                                                                       
             PyYAML: 3.11                                                                        
              ioflo: 1.1.5                                                                       
              PyZMQ: 14.1.1                                                                      
               RAET: 0.4.6                                                                       
                ZMQ: 4.0.4                                                                       
               Mako: 1.0.1                                                                       
                                                                                                 
real    0m1.131s                                                                                 
user    0m0.540s                                                                                 
sys     0m0.112s                                                                                 
```

**Comment by rallytime** [comment]
Thanks for the report @andrejohansson. We'll take a look.

ping @UtahDave

**Comment by basepi** [comment]
This appears to be fixed in the latest 2014.7 and will be fixed in 2014.7.2.

Latest version: https://github.com/saltstack/salt/blob/b37eda267f15c0b436584253506153a5c29448a9/salt/modules/win_file.py#L49

v2014.7.1: https://github.com/saltstack/salt/blob/v2014.7.1/salt/modules/win_file.py#L49

Notice in the latter one the lack of the import of `check_managed_changes`. Someone added that functionality but forgot to import it into the `win_file` stuff.

Thanks for the report!
